lib/raygun/javascript_tracker.rb

```diff
@@ -0,0 +1,46 @@
+# Client for injecting JavaScript code for tracking front end exceptions
```

Comment:
This looks fine. The only other config option I would bother including for now would be the version if it's set